forte_wrapper/allennlp/__init__.py

```diff
@@ -0,0 +1,3 @@
+from forte_wrapper.allennlp.allennlp_processors import MODEL2URL, \
+    AllenNLPProcessor
+from forte_wrapper.allennlp.utils_processor import *
```

Comment:
We only need to expose what we want to. For example, in this module, we want to expose `AllenNLPProcessor`, but we probably don't want to expose `MODEL2URL` and `utils_processor`.